process.py

```diff
@@ -1072,6 +1140,10 @@ def build(release,package):
        jira_url_base = value.strip()
     if key == 'jira_api_key':
        jira_api_key = value.strip()
+    if key == 'gitlab_url':
+       glab_url_base = value.strip()
+    if key == 'gitlab_api_key':
+       glab_api_key = value.strip()
```

Comment:
Don't forget to update the sameple config file with the new keys and reasonable examples of what they should be (of course private keys shouldn't be included, if it's a URL something like http://gitlab.com/YOUR_ID/ca-certificates-fork for instance. You can just use xxxxxxxx for the api_key since it's a random secret string. The sample helps other people set up their system. The actual config file is blocked from git because it has passwords and apikeys that are sensitive.